mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/VeilederJoyrideRoutes.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.mulighetsrommet.api.routes.v1
+
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import no.nav.mulighetsrommet.api.domain.dto.HarFullfortJoyrideRequest
+import no.nav.mulighetsrommet.api.domain.dto.VeilederJoyrideDto
+import no.nav.mulighetsrommet.api.domain.dto.VeilederJoyrideRequest
+import no.nav.mulighetsrommet.api.plugins.getNavIdent
+import no.nav.mulighetsrommet.api.services.VeilederJoyrideService
+import org.koin.ktor.ext.inject
+
+fun Route.veilederJoyrideRoutes() {
+    val veilederJoyrideService: VeilederJoyrideService by inject()
+
+    route("/api/v1/internal/joyride") {
```

Comment:
Kunne evt. definert disse i `VeilederflateRoutes` i stedet for å skille alle routes-filer basert på funksjon?
Riktignok med et annet prefix enn `sanity` :sweat_smile: